content/widgets/smart-sort.md

```diff
@@ -0,0 +1,30 @@
+---
+title: SmartSort
+type: widget
+html: |
+  <div class="ais-SmartSort">
+    <div class="ais-SmartSort-text">
+      <div>We removed some search results to show you the most relevant ones</div>
+    </div>
```

Comment:
```suggestion
    <div class="ais-SmartSort-text">
      We removed some search results to show you the most relevant ones
    </div>
```

Can we change it to like this?

Reply:
RIS will automatically add a `div` if we pass a `TextComponent` but for this example I think so!

Reply:
It's waiting for a `FunctionComponent` so it won't work, should it also accept `strings`? (I've followed the implementation used in `Hits`)

Reply:
<img width="697" alt="image" src="https://user-images.githubusercontent.com/499898/109180248-f5534780-778a-11eb-991b-23fca0d9b20f.png">

string is already allowed IIUC